CONTRIBUTING.md

```
@@ -45,12 +45,12 @@ If you are proposing a feature:
 
 ## Get Started!
 
-Ready to contribute? Here's how to set up `cookiecutter-golang` for local development.
+Ready to contribute? Here's how to set up `cookiecutter-coldbrew` for local development.
 
-1. Fork the `cookiecutter-golang` repo on GitHub.
+1. Fork the `cookiecutter-coldbrew` repo on GitHub.
 2. Clone your fork locally::
 ```bash
-    $ git clone git@github.com:your_name_here/cookiecutter-golang.git
+    $ git clone git@github.com:your_name_here/cookiecutter-coldbrew.git
```

markdownlint-cli2 (0.21.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)